lib/livebook/migration.ex

```diff
@@ -46,6 +47,19 @@ defmodule Livebook.Migration do
     end
   end
 
+  @deploy_key_prefix Livebook.Teams.Requests.deploy_key_prefix()
+
+  defp remove_cli_hub() do
+    # The CLI hub will only be present in the storage if the
+    # user doesn't have the Team hub already persisted with the
+    # user credentials. Consequently, we always remove it and
+    # insert on CLI if applicable.
```

Comment:
I wonder if instead of removing the CLI hub, whenever we run in Agent or CLI mode, we don't save the storage to disk instead? This way it will never persist across runs?

Reply:
So, instead of checking if the hub exists in storage, check if they TeamClient is running?

Reply:
I would not disable it from tests so we can be sure the persistence feature works. :) Unless there is a very good reason to disable during tests too?